### PR TITLE
Wording fixes..

### PR DESCRIPTION
**Pull request by skylarr1227**
fixed many errors in the way it was worded. Will fix more later...

I have never done this before, I hope i did the request here correctly. 
There is more to be corrected... but this is a lot of it. Just trying to help.

### CODE REVIEW

File: Change logs for 3.0.0.md [hunk 1]
```diff
@@ -8,52 +8,52 @@
...
   - For character with 0 rebirths the max level is 5
   - For character with 1 rebirth the Max level is 20
   - For character with 2 to 9 rebirths the max level increases by 5 per rebirth (25, 30, 35...)
   - For character with 10 to 19 rebirths the max level increases by 10 per rebirth (70, 80, 90...)
   - For character with 20 to 38 rebirths the max level increases by 5 per rebirth (170, 175, 180..)
   - For character with more than 38 rebirths the max level will always be 255
-- Characters will stop gaining XP once their character reach max level
-## Items
+- Characters will stop gaining XP once their character reaches max level
+## Items (variety/quality)
```
Comment (Drapersniper) [inline Change logs for 3.0.0.md]: This is a Heading, to break this section of the docs into the Items section, |
`(variety/quality)` doesn't apply here
```suggestion
## Items
```

File: Change logs for 3.0.0.md [hunk 2]
```diff
@@ -8,52 +8,52 @@
...
   - Tinkerer items will stay with you as long as you are a tinkerer
   - Legendary items will stay with you for 3 rebirths after you get them
-     - Say you get a legendary item with you on level 30 Rebirth 3, this item will stay with you until your 6th rebirth
-  - Any loot chests or other items you has prior to rebirthing will be removed
+     - Say you get a legendary item on level 30 Rebirth 3, this item will stay with you until your 6th rebirth
+  - Any loot chests or other items will also be removed
   - You will keep your class upon rebirthing (Say Hello to level 1 Berserkers)
 ### Max Level
-- New dynamic Max levels based on number of rebirths your character has.
+- New dynamic Max level based on your rebirth count.
```
Comment (Drapersniper) [inline Change logs for 3.0.0.md]:
```suggestion
- New dynamic Max level based on number of rebirths your character has.
```

File: Change logs for 3.0.0.md [hunk 3]
```diff
@@ -8,52 +8,52 @@
...
+- When your character reaches its max level simply use the following command: `[p]rebirth`
+- Takes all of your currency 
+- All items will be lost when you rebirth- with the following exceptions
   - Set items will stay with you forever
   - Tinkerer items will stay with you as long as you are a tinkerer
   - Legendary items will stay with you for 3 rebirths after you get them
-     - Say you get a legendary item with you on level 30 Rebirth 3, this item will stay with you until your 6th rebirth
-  - Any loot chests or other items you has prior to rebirthing will be removed
+     - Say you get a legendary item on level 30 Rebirth 3, this item will stay with you until your 6th rebirth
+  - Any loot chests or other items will also be removed
```
Comment (Drapersniper) [inline Change logs for 3.0.0.md]:
```suggestion
  - Any loot chests or other items you had prior to rebirthing will be removed
```

File: Change logs for 3.0.0.md [hunk 4]
```diff
@@ -8,52 +8,52 @@
...
+- Higher chance to encounter stronger monsters that give boosted XP 
+- The ability to convert lesser loot chests into higher quality loot chests
+- Cooldowns for Skills are reduced with each rebirth! 
+note: Users can only trade items to other users with the same or higher number of rebirths.
 #### How to rebirth
-- When your character reach max level they can run `[p]rebirth`
-- This will cost all their current currency
-- This will remove all their items with the following exceptions
+- When your character reaches its max level simply use the following command: `[p]rebirth`
+- Takes all of your currency
```
Comment (Drapersniper) [inline Change logs for 3.0.0.md]:
```suggestion
- This will cost all their current currency
```

**Comment by skylarr1227** [comment]
Well heres an idea, actually make something readable? I cannot even approach this with you. You have gone from respected to childish in moments. I regret ever mentioning, commenting, or using your work

**Pull request closed without merging**